Completion/Unix/Command/_code

```diff
@@ -0,0 +1,42 @@
+#compdef code
+
+local arguments
+
+__code_locales() {
+  local -a locales
+  locales=($(locale -a | sed -e 's/\..*$//'))
+  _describe 'locale' locales
+}
+
+arguments=(
+  {-d,--diff}'[compare two files with each other]:file to compare:_files:file to compare with:_files'
+  {-a,--add}'[add folder(s) to the last active window]:folder:_directories'
+  {-g,--goto}'[open a file at the path on the specified]'
+  {-n,--new-window}'[force to open a new window]'
+  {-r,--reuse-window}'[force to open a file or folder in the last active window]'
+  {-w,--wait}'[wait for the files to be closed before]'
+  '--locale[the locale to use (e.g. en-US or zh-TW)]:locale:__code_locales'
+  '--user-data-dir[specifies the directory that user data is in]:directory:_directories'
```

Comment:
I'd change the verb "specifies" to "specify". Similarly later, I'd change enables to enable, uploads to upload and (un)installs to (un)install. It is more consistent to use the same form for the verbs in descriptions and you have "print", "set" and "list" elsewhere.  Usual zsh convention is this short verb form.

Reply:
Changed lots of verb cases and used “specify” more